src/applications/personalization/search-representative/config/search/searchRepresentativeWidget.jsx

```diff
@@ -1,49 +1,147 @@
+          send your request. You’ll need to mail to your nearest VA regional
+          office or to the service organization or representative that is going
+          to help you.
+        </p>
+        {currentlyShowingData?.map(option => {
           return (
-            <div
-              key={option.id}
-              className="vads-u-background-color--gray-lighter vads-u-padding--1p5 vads-u-margin-bottom--1"
-            >
-              <p className="vads-u-font-size--h3 vads-u-font-family--serif vads-u-font-weight--bold">
-                {option.name}
-              </p>
-              <p>{option.city}</p>
-
-              <button
-                onClick={() => handleClick(option.name)}
-                className="usa-button-secondary"
-              >
-                Choose this representative
-              </button>
-            </div>
+            <>
+              <SearchRepresentativeResult
+                option={option}
+                handleClick={handleClick}
+                key={option.id}
+              />
+            </>
           );
         })}
+        <Pagination
+          onPageSelect={page => handleDataPagination(page)}
+          page={currentPage}
+          pages={pages}
+        />
+        <h2>If you don’t see the representative you want</h2>
+        <p>You can go back and search again</p>
+        <a
+          className="vads-c-action-link--green"
+          href="/view-change-representative/search/representstive-type"
+        >
+          Go back and search again
+        </a>
+        <h2>If you found the representative you want</h2>
+        <p>
+          If you find a representative you ant to help you, contact them by
+          phone, email or mail first to be sure they’re available. If you don’t
+          see the representative you want you want, you can go back and make
+          changes and search again
+        </p>
+        <h3>Contact the representative you want to help you</h3>
+        <p>
+          When you find a VSO or representative you want to help you, contact
+          them first by phone, email or mail first to be sure they’re available.
+        </p>
+        <h3>Let VA know</h3>
+        <p>
+          When you know a VSO or representative is available to help you, there
+          are several ways to make it official.
+        </p>
+        <ul>
+          <li>
+            <p>
+              To have a VSO help you, fill out an Appointment if Veterans
+              Service Organization as Claimant’s Representative (VA form 21-22)
+            </p>
+            <p>
+              <i
+                className="fas fa-download vads-u-padding-right--1 vads-u-color--link-default"
+                aria-hidden="true"
+              />
+              <a
+                className="vads-u-color--link-default"
+                href="/find-forms/about-form-21-22/"
+              >
+                Download VA form 21-22(PDF)
```

Comment:
I think there should be a space before "(PDF)"

Reply:
Added this in.